Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.0
```

Comment:
CC @tomerd / @ktoso I'd personally say 5.3. No new software should be written for 5.2 or below

Reply:
swift-metrics supports 4.2, hence this does too, but I am okay with bumping it here

Reply:
@tomerd / @fabianfett this officially however is major breaking. But if we decide that we ignore that fact, then we should just straight to a very new version ;)

Reply:
@weissi You can not major break, if there has never been a release...

Reply:
If we move directly to `5.2` or `5.3` packages like AHC won't be able to use this package (as a test dependency for example. see #11). Is this a tradeoff you are willing to make?

Reply:
ah that's bad, then 5.1 I guess :P

Reply:
Okay, since AHC requires 5.0, we should target 5.0.